boost-booster-packs/liberty-boost-data-jpa/pom.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
```

Comment:
To start, the parent should be a **TEST** artifact.  If we ultimately decide  app modules  should use our plugin-provided as artifacts at some point..well, that's a significant design decision.  We're not there yet... so the groupId should be something like **boost.maven.it ** and the parent should be in a test group or there should be no parent.